preupg/creator/ui_helper.py

```diff
@@ -128,7 +126,35 @@ def specify_upgrade_path(self):
         return True
 
     def prepare_content_env(self):
-        self.content_path = os.path.join(self.get_group_name(), self.get_content_name())
+        self.content_path = os.path.join(self.get_group_name(),
+                                         self.get_content_name())
+
+    def properties_ini(self):
```

Comment:
- You're right I should rename `properties_ini()`
- I'd say `self.properties_ini_path` cannot be placed in `init` method because it will be only set to right value when `UIHelper` takes `upgrade_path` as argument otherwise there is a `specify_upgrade_path` method which ask user for proper path.
- about moving properties file check into `get_properties_ini_versions`:  I didn't want to add additional logic into that function (just ask user 2 questions).